lua/dap-python.lua

```diff
@@ -55,10 +55,11 @@ function M.setup(adapter_python_path, opts)
   opts = vim.tbl_extend('keep', opts or {}, default_setup_opts)
   dap.adapters.python = function(cb, config)
     if config.request == 'attach' then
+      local port = (config.connect or config).port
       cb({
         type = 'server';
-        port = assert(config.port, '`port` is required for a python `attach` configuration');
-        host = config.host or '127.0.0.1';
+        port = assert(port, '`connect.port` is required for a python `attach` configuration');
+        host = (config.config or config).host or '127.0.0.1';
```

Comment:
@mfussenegger `config.config` should be `config.connect`, I think?

Reply:
Oh right 🤦‍♂️

Thanks for pointing out